lib/std/c.zig

```diff
@@ -39,6 +39,33 @@ pub fn versionCheck(comptime glibc_version: std.SemanticVersion) type {
     };
 }
 
+/// Compatibility function, used by Zig's implementation of libc in `lib/c.zig`.
+/// Assumes that `a` and `b` are multi-item pointers with NUL-termination.
+/// Returns `-1` if `a` appears before `b` in lexicographical order.
+/// Returns `1` if `a` appears after `b` in lexicographical order.
+/// Returns `0` if `a` and `b` are equal.
+/// See also `std.ascii.orderIgnoreCase` for idiomatic function
+/// with parameters as Zig's slices.
```

Comment:
This comment is incorrect, `std.mem.order()` is the idiomatic Zig equivalent.

`std.ascii.orderIgnoreCase()` has different behavior.

Reply:
done